plugins/woocommerce-admin/client/products/sections/pricing-section.tsx

```diff
@@ -123,7 +124,6 @@ export const PricingSection: React.FC = () => {
 				<CardBody>
 					<BaseControl
 						id="product_pricing_regular_price"
-						className={ regularPriceProps?.className ?? '' }
```

Comment:
I'm okay with this change. As mentioned before, ideally we should abstract away this `<BaseControl>` usage so we aren't directly using it here. That is for the future though.